bigquery_etl/run_multipart_query.py

```diff
@@ -90,23 +93,16 @@ def dirpath(string):
 )
 parser.add_argument(
     "--schema_update_option",
+    action="append",
+    choices=[
+        bigquery.SchemaUpdateOption.ALLOW_FIELD_ADDITION,
+        bigquery.SchemaUpdateOption.ALLOW_FIELD_RELAXATION,
+    ],
+    default=[],
+    dest="schema_update_options",
```

Comment:
I'm wondering if we should rename `schema_update_option` to `schema_update_options` here so that it is consistent with the parameter naming in `bigquery.QueryJobConfig`. Doing that would require updating telemetry-airflow though.

Reply:
I think we should keep it as-is to match the `bq` cli tool.